packages/generator/package.json

```diff
@@ -21,7 +22,9 @@
     "p-queue": "^6.4.0",
     "physical-cpu-count": "^2.0.0",
     "prettier": "^2.0.5",
-    "protocol-buffers": "^4.1.2"
+    "protocol-buffers": "^4.1.2",
+    "ts-node": "^10.9.2",
+    "typescript": "^5.8.2"
   },
   "devDependencies": {
     "delay-async": "^1.2.0"
```

Comment:
```suggestion
    "@types/fs-extra": "^11.0.4",
    "delay-async": "^1.2.0"
```
Could you add `@types/fs-extra` as a dev dep? TS is complaining that it is being imported without declaration files.

<img width="1518" alt="image" src="https://github.com/user-attachments/assets/f0640988-b5f9-486b-b1dd-58a0aeb054b9" />